build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
 	id 'com.github.kt3k.coveralls' version '2.6.3'
+ id "org.sonarqube" version "2.5"
```

Comment:
It works for the sonarqube I am running on my local machine. I oversaw that it was still in the build.gradle when merging the branches. I normally to not commit this change because I only use it locally.